File: tests/fixtures/anti-slop/acceptance/csharp/RoundTrip.cs
using System;

public sealed class Item
{
    public void Read() { }
}

public static class RoundTrip
{
    public static Item ObjectCastRoundTrip(Item value)
    {
        return (Item)(object)value;
    }

    public static System.Reflection.MethodInfo LiteralReflection()
    {
        return typeof(Item).GetMethod("Read")!;
    }

    public static Item DirectValue(Item value)
    {
        return value;
    }
}

File: tests/fixtures/anti-slop/acceptance/java/RoundTrip.java
final class Item {
    int read() {
        return 1;
    }
}

public class RoundTrip {
    static Item objectCastRoundTrip(Item value) {
        return (Item) (Object) value;
    }

    static java.lang.reflect.Method literalReflection() throws Exception {
        return Item.class.getDeclaredMethod("read");
    }

    static Item directValue(Item value) {
        return value;
    }
}
